controllers/usersignup/approval_test.go

```diff
@@ -60,6 +59,60 @@ func TestGetClusterIfApproved(t *testing.T) {
 		assert.Equal(t, "member2", clusterName.getClusterName())
 	})
 
+	t.Run("with two clusters available, the second one has required cluster-role label", func(t *testing.T) {
+		// given
+		signup := commonsignup.NewUserSignup()
+		toolchainConfig := commonconfig.NewToolchainConfigObjWithReset(t,
+			testconfig.AutomaticApproval().
+				Enabled(true),
+			testconfig.CapacityThresholds().
+				MaxNumberOfSpaces(testconfig.PerMemberCluster("member1", 1000), testconfig.PerMemberCluster("member2", 1000)).
+				ResourceCapacityThreshold(80, testconfig.PerMemberCluster("member1", 70), testconfig.PerMemberCluster("member2", 75)))
```

Comment:
for the simplicity of the tests, you can drop the capacity thresholds. 
```suggestion
				Enabled(true))
```

Reply:
removed in [6b01926](https://github.com/codeready-toolchain/host-operator/pull/736/commits/6b01926137a7f5c783ed0b886e61eaba8cc13642)